BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeDeepLinkActivity.kt

```diff
@@ -0,0 +1,17 @@
+package com.braintreepayments.api
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+
+/**
+ * This Activity is used by the SDK Activity to handle deep links on behalf of the merchant
+ * It should not be used directly.
+ */
+class BraintreeDeepLinkActivity : AppCompatActivity() {
+    private val browserSwitchClient = BrowserSwitchClient()
+    override fun onCreate(savedInstanceState: Bundle?) {
```

Comment:
Which Kotlin style guide are we currently using? We should pick one and pin it for now for consistency across PPCP & BT.

Reply:
We're using the JetBrains official [Kotlin style guide](https://kotlinlang.org/docs/coding-conventions.html), but we should definitely draft our own. Should we place our mobile coding conventions in a single repo to avoid duplication y'all think?

Reply:
Great idea! Ideally one hosted at `github.com/paypal` which can apply across products and repos

Reply:
Created a ticket for this: DTNOR-967